packages/react-conformance/src/defaultTests.tsx

```diff
@@ -146,6 +146,64 @@ export const defaultTests: TestObject = {
     });
   },
 
+  /** Component file handles classname prop */
+  'component-handles-classname': (componentInfo: ComponentDoc, testInfo: IsConformantOptions) => {
+    const { Component, wrapperComponent, helperComponents = [], requiredProps, customMount = mount } = testInfo;
+
+    let el = customMount(<Component {...requiredProps} />);
+    let component = getComponent(el, helperComponents, wrapperComponent);
+
+    const defaultClassNames =
+      component
+        ?.getDOMNode()
+        ?.getAttribute('class')
+        ?.split(' ') || [];
```

Comment:
I am a bit worried about this check...

Our components are using CSS-in-JS renderer and for N* we have a noop renderer when there is no React Context provided. Is it the same for V8/next? For example, are there any guarantees that we will not get `msComponent-root-XXX` in in `defaultClassNames`?

Reply:
I believe that it should be fine. In scenarios where a defaultClassName isn't recieved to begin with. The test should still pass as it will only run if one is provided:
```jsx
if (defaultClassNames.length && defaultClassNames[0] !== '') {
...
}
``` 
Otherwise, it will test to see if the existing defaultClassName exists alongside the testClassName.